docs/tutorials/config.md

```diff
@@ -395,6 +395,7 @@ lr_config = dict(  # Learning rate scheduler config used to register LrUpdater h
     0.001,  # The ratio of the starting learning rate used for warmup
     step=[8, 11])  # Steps to decay the learning rate
 total_epochs = 12  # Total epochs to train the model
```

Comment:
This line should be removed or should be indicated as unnecessary in the comments.